freespace2/freespace.cpp

```diff
@@ -2752,16 +2634,15 @@ void do_timing_test(float frame_time)
 	static float test_time = 0.0f;
 
 	static sound_handle snds[NUM_MIXED_SOUNDS];
-	int i;
 
 	if ( test_running ) {
 		framecount++;
 		test_time += frame_time;
 		if ( framecount >= NUM_FRAMES_TEST ) {
 			test_running = 0;
 			nprintf(("General", "%d frames took %.3f seconds\n", NUM_FRAMES_TEST, test_time));
-			for ( i = 0; i < NUM_MIXED_SOUNDS; i++ )
-				snd_stop(snds[i]);
+			for (auto snd : snds)
```

Comment:
Why is this not using `auto &`?

Reply:
sound_handle is a strong type wrapper around an int, so pass by value would generally be preferable, in the rare cases, where it makes a difference.
But I see that in this context it is irrelevant in case of generated code, and not apparent to a reader.

I'll make it auto& if you prefer that.

(I personally like to use value instead of reference where possible, but will happily follow conventions on the project.)

Reply:
I don't have a preference; I was asking for clarification.